cfg/cfg.go

```diff
@@ -102,4 +114,8 @@ func Print() {
 	if UseBearerToken {
 		logrus.Infof("==> Authenticated user ID: %s", AuthenticatedUserID)
 	}
+	if IdleTimeout > 0 {
```

Comment:
Do we also need to check stop-retry-period as well to make sure it's not negative?

Reply:
I suppose stop-retry-period must not be negative. Will add validation.